### ISSUE TITLE
Error... [TLS?]

### ISSUE THREAD

**Issue by NavyCo** [hydrabolt/discord.js#506]
```
RangeError: out of range index
    at RangeError (native)
    at fastCopy (C:\Users\NicoP\Desktop\_\Discord-Bot\node_modules\ws\lib\Receiver.js:382:24)
    at Receiver.add (C:\Users\NicoP\Desktop\_\Discord-Bot\node_modules\ws\lib\Receiver.js:82:3)
    at TLSSocket.firstHandler (C:\Users\NicoP\Desktop\_\Discord-Bot\node_modules\ws\lib\WebSocket.js:774:22)
    at emitOne (events.js:96:13)
    at TLSSocket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:177:18)
    at TLSSocket.Readable.push (_stream_readable.js:135:10)
    at TLSWrap.onread (net.js:542:20)
```

**Comment by abalabahaha** [comment]
Was the bot doing anything at the time?

Are all your modules updated?

What version of Node are you on?

**Comment by NavyCo** [comment]
Version: Just updated the same day I updated.
Modules: All updated
Actions: Was not doing anything actually, just being idle.

**Comment by hydrabolt** [comment]
Should be fixed in [9.0.2](https://github.com/hydrabolt/discord.js/releases/tag/9.0.2) (as a side note, make sure `ws` is actually up-to-date!)
